pkg/v1/tkg/client/client.go

```diff
@@ -67,7 +68,6 @@ type InitRegionOptions struct {
 	InfrastructureProvider      string
 	ControlPlaneProvider        string
 	Namespace                   string
-	WatchingNamespace           string
```

Comment:
Can we note in the PR description that we're removing this (and why)?

Reply:
Done